### ISSUE TITLE
Issue with CONTENTALIGNMENT

### ISSUE THREAD

**Issue by valinet** [nptr/msstyleEditor#43]
Hi

First off, thank you very much for this utility, much needed as a free and well working alternative for hacking a bit with msstyles.

Now, I tried making Explorer title bars have their caption centered. For that, a trick that indeed works is described here: https://www.sevenforums.com/customization/186924-center-titlebar-text.html. Basically, one has to add CONTENTALIGNMENT property to Window\CAPTION and Window\MAXCAPTION. All works well, but, unfortunately, there is a small issue: if you want the title bar text to be centered, you have to set the property to Right. Similarly, if you want to have it aligned to the right, the property needs to be set to Center.

The way I see it, these could be caused by two things:
* you simply interpreted the values in the enum wrong, and labelled them switched in your application's code; this should be an easy fix, in VisualStyleEnums.cpp
* Microsoft interprets the values wrong in their File Explorer code, which you can do nothing about, and this is not an issue with your application at all

Thanks again for this wonderful application.

**Comment by nptr** [comment]
Thanks for the feedback and glad you like my application!

It was cause no. 1 - mixed up the enums. I made a build with the fix and uploaded
it as pre-release version [1.4.0.7](https://github.com/nptr/msstyleEditor/releases/1.4.0.7).

**Comment by valinet** [comment]
I am glad that you figured it out, thank you very much.

I looked a bit more on the source code, really nice, and the app proved very useful for my use case. Thanks again.

On my part, I checked the uploaded pre-release binary (1.4.0.7). From the brief check I have done, it looks that the bug is indeed fixed, i.e. the CONTENTALIGNMENT property is labelled properly. I propose closing this issue here, as well.
